cmd/ebrelayer/siflogger/logger.go

```diff
@@ -0,0 +1,233 @@
+package siflogger
```

Comment:
I think we can create a separate package for the logger interface , instead of putting it under ebrelayer ?

Reply:
yes, @utx0 could you please advise name of package and folder location for package files.

Reply:
`pkg/siflogger`